agent-tooling/src/main/java/io/opentelemetry/auto/tooling/VersionLogger.java

```diff
@@ -28,9 +28,10 @@
   /** Log version string for java-agent */
   public static void logAllVersions() {
     log.info(
-        "opentelemetry-auto - version: {}",
+        "opentelemetry-javaagent - version: {}",
         getVersionString(
-            ClassLoader.getSystemClassLoader().getResourceAsStream("opentelemetry-auto.version")));
+            ClassLoader.getSystemClassLoader()
+                .getResourceAsStream("opentelemetry-javaagent.version")));
```

Comment:
Just a thought while we're here what do you think about moving the file to a package?

Reply:
It's a good thought, I opened #669 to discuss/track.